diff --git a/node_modules/@cosmjs/ledger-amino/build/launchpadledger.d.ts b/node_modules/@cosmjs/ledger-amino/build/launchpadledger.d.ts
index c4a2bab..3149514 100644
--- a/node_modules/@cosmjs/ledger-amino/build/launchpadledger.d.ts
+++ b/node_modules/@cosmjs/ledger-amino/build/launchpadledger.d.ts
@@ -9,6 +9,7 @@ export interface LaunchpadLedgerOptions {
     readonly hdPaths?: readonly HdPath[];
     readonly prefix?: string;
     readonly testModeAllowed?: boolean;
+    readonly ledgerAppName?: string;
 }
 export declare class LaunchpadLedger {
     private readonly testModeAllowed;
diff --git a/node_modules/@cosmjs/ledger-amino/build/launchpadledger.js b/node_modules/@cosmjs/ledger-amino/build/launchpadledger.js
index 27173da..4866e83 100644
--- a/node_modules/@cosmjs/ledger-amino/build/launchpadledger.js
+++ b/node_modules/@cosmjs/ledger-amino/build/launchpadledger.js
@@ -19,15 +19,17 @@ const cosmosBech32Prefix = "cosmos";
 const requiredCosmosAppVersion = "1.5.3";
 class LaunchpadLedger {
     constructor(transport, options = {}) {
-        var _a, _b, _c;
+        var _a, _b, _c, _d;
         const defaultOptions = {
             hdPaths: [cosmosHdPath],
             prefix: cosmosBech32Prefix,
             testModeAllowed: false,
+            ledgerAppName: ''
         };
         this.testModeAllowed = (_a = options.testModeAllowed) !== null && _a !== void 0 ? _a : defaultOptions.testModeAllowed;
         this.hdPaths = (_b = options.hdPaths) !== null && _b !== void 0 ? _b : defaultOptions.hdPaths;
         this.prefix = (_c = options.prefix) !== null && _c !== void 0 ? _c : defaultOptions.prefix;
+        this.ledgerAppName = (_d = options.ledgerAppName) !== null && _d !== void 0 ? _d : defaultOptions.ledgerAppName;
         this.app = new ledger_cosmos_js_1.default(transport);
     }
     async getCosmosAppVersion() {
@@ -87,8 +89,8 @@ class LaunchpadLedger {
         if (appName.toLowerCase() === `dashboard`) {
             throw new Error(`Please open the Cosmos Ledger app on your Ledger device.`);
         }
-        if (appName.toLowerCase() !== `cosmos`) {
-            throw new Error(`Please close ${appName} and open the Cosmos Ledger app on your Ledger device.`);
+        if (this.ledgerAppName && appName.toLowerCase() !== this.ledgerAppName.toLowerCase()) {
+            throw new Error(`Please close ${appName} and open the ${this.ledgerAppName} Ledger app on your Ledger device.`);
         }
     }
     async verifyDeviceIsReady() {
