diff --git a/node_modules/@cosmjs/stargate/build/signingstargateclient.js b/node_modules/@cosmjs/stargate/build/signingstargateclient.js
index 5df70e4..c704c31 100644
--- a/node_modules/@cosmjs/stargate/build/signingstargateclient.js
+++ b/node_modules/@cosmjs/stargate/build/signingstargateclient.js
@@ -30,6 +30,7 @@ exports.defaultRegistryTypes = [
     ["/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward", tx_2.MsgWithdrawDelegatorReward],
     ["/cosmos.distribution.v1beta1.MsgWithdrawValidatorCommission", tx_2.MsgWithdrawValidatorCommission],
     ["/cosmos.gov.v1beta1.MsgSubmitProposal", tx_3.MsgSubmitProposal],
+    ["/cosmos.gov.v1beta1.MsgDeposit", tx_3.MsgDeposit],
     ["/cosmos.gov.v1beta1.MsgVote", tx_3.MsgVote],
     ["/cosmos.staking.v1beta1.MsgBeginRedelegate", tx_4.MsgBeginRedelegate],
     ["/cosmos.staking.v1beta1.MsgCreateValidator", tx_4.MsgCreateValidator],
@@ -63,6 +64,7 @@ class SigningStargateClient extends stargateclient_1.StargateClient {
     constructor(tmClient, signer, options) {
         super(tmClient);
         const { registry = createDefaultRegistry(), aminoTypes = new aminotypes_1.AminoTypes({ prefix: options.prefix }) } = options;
+        console.log(registry)
         this.registry = registry;
         this.aminoTypes = aminoTypes;
         this.signer = signer;
