diff --git a/node_modules/@cosmjs/proto-signing/build/signing.d.ts b/node_modules/@cosmjs/proto-signing/build/signing.d.ts
index 0c43f4d..0ed847a 100644
--- a/node_modules/@cosmjs/proto-signing/build/signing.d.ts
+++ b/node_modules/@cosmjs/proto-signing/build/signing.d.ts
@@ -10,6 +10,6 @@ import { Any } from "cosmjs-types/google/protobuf/any";
 export declare function makeAuthInfoBytes(signers: ReadonlyArray<{
     readonly pubkey: Any;
     readonly sequence: number;
-}>, feeAmount: readonly Coin[], gasLimit: number, signMode?: SignMode): Uint8Array;
+}>, feeAmount: readonly Coin[], gasLimit: number, signMode?: SignMode, granter?: string): Uint8Array;
 export declare function makeSignDoc(bodyBytes: Uint8Array, authInfoBytes: Uint8Array, chainId: string, accountNumber: number): SignDoc;
 export declare function makeSignBytes({ accountNumber, authInfoBytes, bodyBytes, chainId }: SignDoc): Uint8Array;
diff --git a/node_modules/@cosmjs/proto-signing/build/signing.js b/node_modules/@cosmjs/proto-signing/build/signing.js
index 96d86f5..9249e63 100644
--- a/node_modules/@cosmjs/proto-signing/build/signing.js
+++ b/node_modules/@cosmjs/proto-signing/build/signing.js
@@ -26,12 +26,13 @@ function makeSignerInfos(signers, signMode) {
  *
  * This implementation does not support different signing modes for the different signers.
  */
-function makeAuthInfoBytes(signers, feeAmount, gasLimit, signMode = signing_1.SignMode.SIGN_MODE_DIRECT) {
+function makeAuthInfoBytes(signers, feeAmount, gasLimit, signMode = signing_1.SignMode.SIGN_MODE_DIRECT, granter) {
     const authInfo = {
         signerInfos: makeSignerInfos(signers, signMode),
         fee: {
             amount: [...feeAmount],
             gasLimit: long_1.default.fromNumber(gasLimit),
+            granter
         },
     };
     return tx_1.AuthInfo.encode(tx_1.AuthInfo.fromPartial(authInfo)).finish();
